test/metabase/api/search_test.clj

```diff
@@ -474,6 +466,62 @@
                (map #(select-keys % [:name :model :description])
                     (search-request-data-with sorted-results :crowberto :q "aviaries"))))))))
 
+(deftest indexed-entity-test
+  (testing "Should search indexed entities"
+    (mt/dataset airports
+      (let [query (mt/mbql-query municipality)]
+        (mt/with-temp* [Card [model {:dataset       true
```

Comment:
Could use `mt/with-temp`.

Reply:
I don't like `mt/with-temp`. Since if you need to add another temp item the syntax changes.

Reply:
> I don't like `mt/with-temp`. Since if you need to add another temp item the syntax changes.

I can understand that. I think the mere presence of `mt/with-temp` is causing some confusion. It would be better if it were private.